### ISSUE TITLE
require-hook: false warning for test.fixme

### ISSUE THREAD

**Issue by EvgenyOrekhov** [playwright-community/eslint-plugin-playwright#249]
```ts
test.fixme('some test', async ({ page }) => {
  // ...
});
```

Expected: no warning.

Actual: "This should be done within a hook" warning

eslint-plugin-playwright version: 1.3.1
Node.js version: 18.19.0

**Comment by mskelton** [comment]
This is already fixed, just waiting on a release.
